app/server/appsmith-server/src/main/java/com/appsmith/server/constants/Constraint.java

```diff
@@ -0,0 +1,6 @@
+package com.appsmith.server.constants;
+
+public class Constraint {
+    public static final int ORGANIZATION_LOGO_CHUNKS = 63;
+    public static final int ORGANIZATION_LOGO_SIZE = 250;
```

Comment:
```suggestion
    public static final int ORGANIZATION_LOGO_SIZE_KB = 250;
```

Adding the units in the constant makes it super clear what we are talking about.